pkgs/applications/misc/slstatus/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchgit, pkgconfig, writeText, libX11, conf ? null }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "slstatus-${version}";
+  version = "unstable-2018-03-21";
+
+  src = fetchgit {
+    url = https://git.suckless.org/slstatus;
+    rev = "e79d4932ea413c277425ff92c456741e0730c3d6";
+    sha256 = "1c1al3vkywv0rpg4qaa67wvnc1v5ggvlsc1ynq11cr8gs4msvcl0";
+  };
+
+  configFile = optionalString (conf!=null) (writeText "config.def.h" conf);
+  preBuild = optionalString (conf!=null) "cp ${configFile} config.def.h";
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ libX11 ];
+
+  installPhase = "make install PREFIX=$out";
```

Comment:
You can remove this line and instead use `installFlags = [ "PREFIX=$(out)" ]`

Reply:
Thanks and please excuse my late reply.